sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -22,8 +20,22 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
     internal static class LogsHelper
     {
         private const int Version = 2;
-        private static readonly ConcurrentDictionary<int, string> s_depthCache = new ConcurrentDictionary<int, string>();
-        private static readonly Func<int, string> s_convertDepthToStringRef = ConvertDepthToString;
+        private static readonly Action<LogRecordScope, IDictionary<string, string>> s_processScope = (scope, properties) =>
+        {
+            foreach (KeyValuePair<string, object?> scopeItem in scope)
+            {
+                if (string.IsNullOrEmpty(scopeItem.Key) || scopeItem.Key == "{OriginalFormat}")
+                {
+                    continue;
+                }
+
+                if (scopeItem.Value != null)
+                {
+                    properties.Add(scopeItem.Key,
```

Comment:
Needs truncate here: https://github.com/Azure/azure-sdk-for-net/blob/e5af5510abe1bde5df2df38140167ffc2fdcab93/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/SchemaConstants.cs#L63-L64

Not a blocker for this PR.

Reply:
Also, this may throw in case of duplicates. Need try catch

Reply:
can scopeItem contain duplicate keys or it is flattened at SDK?

Reply:
Duplicates can exist within scope. Also a property on formatted log can have the same name as scope property. We should handle this case [here](https://github.com/Azure/azure-sdk-for-net/blob/e5af5510abe1bde5df2df38140167ffc2fdcab93/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs#L66) as well.

Reply:
SDK do not dedup

Reply:
Fixed dc9f3376dfb569d744c342e19aa06a874424f2e0